modules/cli-manager-about.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/cli_manager/cli-manager-install.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cli-manager-about_{context}"]
+= About the {cli-manager}
+
+The {cli-manager} uses Krew to make installing and updating cli plugins easier. It can be used in both connected and disconnected environments, but has particular utility in disconnected environments. Cluster administrators can add CLI plugins to the {cli-manager} and users can then add CLI plugins when needed regardless of whether or not the environment is disconnected.
```

Comment:
I think it's confusing using "add plugins" both when talking about admins making plugins available, and also when users add/install them locally.

I'd try to figure out precise terminology for each of these tasks, then update them throughout the docs.

Perhaps that the admin adds the plugins, and then it's install/uninstall when we're talking about users actually installing them to use